ang/crmMailing/BlockMailing.html

```diff
@@ -79,5 +79,19 @@
         placeholder="Subject"
         name="subject" />
     </div>
+    <div ng-if="crmMailingConst.isMultiLingual">
+      <div crm-ui-field="{name: 'subform.language', title: ts('Language')}">
+        <select
+          crm-ui-id="subform.language"
+          crm-ui-select="{dropdownAutoWidth : true, allowClear: false}"
+          name="language"
+          ng-model="mailing.language"
+          required
+          >
+          <option value=""></option>
```

Comment:
I think we should remove this empty ```option``` as it look odd to have a blank placeholder. 
![image](https://cloud.githubusercontent.com/assets/3735621/19148772/ea2d4e36-8bdb-11e6-923b-834d1f1f2b66.png)

Since it is a required field so we have to choose any of the enabled languages listed, so what if we remove this blank option OR place a sensible placeholder say ```- choose language -``` !!